src/backends/onnx/autofill.cc

```diff
@@ -299,6 +299,10 @@ AutoFillOnnx::Create(
   std::set<std::string> version_dirs;
   RETURN_IF_ERROR(GetDirectorySubdirs(model_path, &version_dirs));
 
+  // There must be at least one version directory that we can inspect to
+  // attempt to determine the platform. For now we allow multiple versions
+  // and only inspect the first verison directory to ensure it is valid.
```

Comment:
Not true for ONNX, see code below.

Reply:
@CoderHam You should just remove this comment... the ONNX code is ok as is.

Reply:
Sure. Will do.